lib/audit/formIds.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const glob = require('tiny-glob');
+const yaml = require('js-yaml');
+
+const id = /(id:\ ?)(\w*)/;
+
+module.exports = {
+    audit: async () => {
+        let results = {};
+
+        // Collect ids.
+        const schemaFiles = await glob('{app,modules}/**/model_schemas/*.yml', { filesOnly: true });
```

Comment:
Legacy dir is "custom_model_types" or something like that

@Slashek will remember ;)